### PR TITLE
Add labels in model config

### PR DESCRIPTION
**Pull request by kateinkim**
### What's changed
Per issue #46, the `CLASSES` from `labels.ts` is going to be used as an optional label prop in `ModelConfig`.

### examples
The inference data in Storybook actions will be returned like so : 

1. Classfication model
CleanShot 2022-01-14 at 14 39 13@2x

2. SSD model with labels
CleanShot 2022-01-14 at 14 39 29@2x

3. SSD model without labels
CleanShot 2022-01-14 at 14 40 26@2x

### CODE REVIEW

File: packages/ai-lab/src/lib/helpers.tsx [hunk 1]
```diff
@@ -133,12 +134,27 @@ export async function getModelDetections(
...
+  results: number[] | Detections,
+  modelConfig: ModelConfig = defaultModelConfig
+) {
+  // setting CLASSES as default label. Without it ssdInferData will not return any labels in the Inference data results.
+  const { labels = CLASSES } = modelConfig;
+
+  if (Array.isArray(results)) {
+    const res = results.map((data, key) => ({
+      class: key,
+      classLabel: `class ${key}`,
```
Comment (GantMan) [inline packages/ai-lab/src/lib/helpers.tsx:147]: why aren't we using labels here?

File: packages/ai-lab/src/components/AILabObjectDetectionUI.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import * as tf from '@tensorflow/tfjs';
 import React, { useEffect, useRef } from 'react';
 import { ObjectDetectionUIProps } from '../types';
-import { CLASSES } from '../lib/labels';
+import { CLASSES } from './labels';
```
Comment (GantMan) [inline packages/ai-lab/src/components/AILabObjectDetectionUI.tsx:4]: How is this supporting custom labels?  It's hardcoded

**Review by GantMan — CHANGES_REQUESTED** [review]
Your images in your PR look fanatstic!  But.... where are those storybook changes?   I don't see any storybook files modified in this PR?   The purpose of storybook is to supply demos.  I expected to see your code changes demonstrated in there.  Did you modify it for the screenshots and then revert?

One big part here, is you've kept labels inside the components.  They should be out of the npm lib, like the model file is outside.  But you're loading:
https://github.com/infinitered/ai-lab/blob/move-labels/packages/ai-lab/src/components/labels.ts

What if I wanted to do an SSD model with labels in spanish?  You've hardcoded the labels in the `AILabObjectDetectionUI.tsx` file.  

I'm not sure this PR really applies the feature.   Basically all labels should come in from config, and shouldn't exist inside the npm lib at all.  Let me know if you need me to clarify on any part of that, happy to explain further.

File: packages/ai-lab/src/lib/helpers.tsx (hunk 1 above)
Comment (kateinkim) [reply packages/ai-lab/src/lib/helpers.tsx:147]: I was under the impression that the classification models don't need labels and it was only an option for the SSD model that would need labels optionally.

File: packages/ai-lab/src/components/AILabObjectDetectionUI.tsx (hunk 2 above)
Comment (kateinkim) [reply packages/ai-lab/src/components/AILabObjectDetectionUI.tsx:4]: I thought I'd still need to use it as default labels in order to match the data for now. I didn't think the ability to support custom labels was going to be part of this PR.

File: packages/ai-lab/src/lib/helpers.tsx (hunk 1 above)
Comment (GantMan) [reply packages/ai-lab/src/lib/helpers.tsx:147]: Nah, classification models def should have labels, too.   For instance NSFWJS has 5 classes, and 5 labels.

All detection models will use labels, and should apply the labels that fit that model.

File: packages/ai-lab/src/components/AILabObjectDetectionUI.tsx (hunk 2 above)
Comment (GantMan) [reply packages/ai-lab/src/components/AILabObjectDetectionUI.tsx:4]: I understand what you're saying.  The task of making labels come in from modelconfig, should mean that the labels passed into modelconfig should be used here, too.   I get breaking it into steps, but I think it's best to fully connect the modelconfig labels here.

Comment (kateinkim) [reply packages/ai-lab/src/components/AILabObjectDetectionUI.tsx:4]: Requested changes : 
1. Classification model also accepts labels optionally.

https://user-images.githubusercontent.com/53795920/149607203-314e6c72-46fb-4039-94c3-febc6a0569fd.mp4

2. Both models are supporting custom labels in storybook.

https://user-images.githubusercontent.com/53795920/149607247-bc8d93a4-864d-4a14-8f37-fd2a99f23643.mp4

**Review by GantMan — APPROVED** [review]
You rock!

**Pull request merged**